.github/workflows/soroban-rpc.yml

```diff
@@ -32,6 +32,11 @@ jobs:
       - run: |
           rustup update
       - run: make build-libpreflight
+      - name: Build soroban contract fixtures
```

Comment:
no need to `rustup update` again. why don't you add the `rustup target add wasm32-unknown-unknown` to the step before ?


 Also, consider adding the `make build-test-wasms` to the previous step.